e2e/testdata/fn-render/out-of-place-fnchain-stdout-results/.expected/config.yaml

```diff
@@ -1,3 +1,5 @@
+stdErr: "For complete results, see"
```

Comment:
I think we should omit beginning of output like this, otherwise, it's not really testing what we want.

This is not the expected sttderror from:

```
kpt fn render -o stdout --results-dir $results \| kpt fn eval - --image gcr.io/kpt-fn/set-annotations:v0.1.3 --results-dir $results -- foo=bar \| kpt fn eval - --image gcr.io/kpt-fn/set-labels:v0.1.3 --results-dir $results -- tier=backend
```

Reply:
This is to test if the substring we expect is present in stderr and not checking the entire stderr. Not sure if there is a better way to refer the temporary directory created in exec.sh.

Reply:
Updated stderr. But it will include the progress of render only.

Reply:
This is little tricky because in each of the command `structured-result-file` will depend on the temporary directory, so you can do only partial assert here and in this case, substring match only for the first command. 

test harness can be extended to support make temporary dir etc. available in a templatized form, but not sure if we want to go there.

But we should be able to test `piped` version without `structured-result` for non-structured CLI output. So, lets have a version of this test without structured result just for testing the presentation output. That should be sufficient.

Reply:
That is covered as part of other tests. This test is to test the behavior when the resources are piped and results are written to --results-dir. I updated the test, it will test the render result which is good enough IMO. Please look at updated file.